WordPress/src/main/java/org/wordpress/android/models/JetpackPoweredScreen.kt

```diff
@@ -18,24 +17,12 @@ sealed interface JetpackPoweredScreen {
     @Parcelize
     enum class WithStaticPoster(
         val screen: WithDynamicText,
-        @AnimRes val animResLtr: Int,
-        @AnimRes val animResRtl: Int,
```

Comment:
@ovitrif I've removed the annotation to avoid a warning `Expected resource of type anim` that I couldn't work around without suppressing.  I'll be happy to follow another approach on this :)

Reply:
Great suggestion @ovitrif 👍 
I've updated it with 27fcd9025d280be9198e6b8d94ad8cce75d6b6f6

Reply:
Thanks a lot, looks even more shiny now 🚀 🥇 🙇🏻 !